.github/workflows/go.yml

```diff
@@ -7,73 +7,66 @@ on:
     branches: "*"
 
 jobs:
-
   build:
     name: ${{ matrix.os }} - Go ${{ matrix.go_version }}
     runs-on: ubuntu-latest
     strategy:
-     # If you want to matrix build , you can append the following list.
+      # If you want to matrix build , you can append the following list.
       matrix:
         go_version:
-          - 1.13
-          - 1.14
+          - 1.18
+          - 1.19
```

Comment:
As a component/sdk, code should meet the requirements of most versions.
In generally, versions of develop language are backward compatible, so we should make sure sentinel can be build as low as go-version.
And even if the go-version is incompatible, we also need to upgrade to a smoothed version, rather then upgrade to the latest version.